### PR TITLE
Featureplay vs computer

### PR DESCRIPTION
**Pull request by andrewljohnson**

### CODE REVIEW

**Comment by andrewljohnson** [comment]
```
➜  midrange go install ./... && play

 ~~~~~~ Welcome to Rogue ~~~~~~

1) Human vs AI
2) AI vs AI

Enter a number:
```

File: game/player.go [hunk 1]
```diff
@@ -29,10 +29,12 @@ func NewPlayer(deck *Deck) *Player {
 }
 
 func (p *Player) Draw() {
-	p.AddToHand(p.Deck.Draw())
+	card := p.Deck.Draw()
+	p.AddToHand(card)
 }
 
 func (p *Player) AddToHand(c *Card) {
+	// TODO this seems like suspicious code, why would a card be nil here?
```
Comment (lacker) [inline game/player.go]: A card would be nil here if you attempted to start a game with less than 7 cards in your deck. :P

File: game/player.go [hunk 2]
```diff
@@ -43,7 +45,7 @@ func (p *Player) AddToHand(c *Card) {
 func (p *Player) AvailableMana() int {
 	answer := 0
 	for _, card := range p.Board {
-		if card.IsLand {
+		if card.IsLand && card.Tapped == false {
```
Comment (lacker) [inline game/player.go]: typically one just says "!card.Tapped" rather than "card.Tapped == false"

**Comment by lacker** [comment]
two nitpicks, feel free to push when ready

**Pull request merged**